app/controllers/opportunity_matches_controller.rb

```diff
@@ -8,14 +8,9 @@ class OpportunityMatchesController < ApplicationController
 
   def index
     clients_for_route = Client.available_for_matching(@opportunity.match_route)
-    @matches = @opportunity.matching_clients(clients_for_route).page(params[:page]).per(25)
-    @sub_program = @opportunity.sub_program
-    @program = @sub_program.program
-  end
-
-  def all
-    clients_for_route = Client.all
-    @matches = @opportunity.matching_clients(clients_for_route).page(params[:page]).per(25)
+    @actives = @opportunity.active_matches.map { |match| match.client }
+    @availables = @opportunity.matching_clients(clients_for_route)
+    @matches = Kaminari.paginate_array(@actives + @availables).page(params[:page]).per(25)
```

Comment:
I don't see a reason to paginate this if we have to load them all anyway.  We should test what happens when 200 clients match though.  Might make the page very slow.

Reply:
I experimented with using a back port of Rails 5 or to combine the actives and available, but a limitation of that functionality is the two sets must differ only by the where clauses...

Also experimented with plucking the ids for the two sets and then reifying the objects based on those, but couldn't figure out how to order the results... Currently, the active matches appear at the top, but maybe that isn't required, the table could just be ranked by the route, and highlight the active matches?